### ISSUE TITLE
Unable to serve /agent/13911518246308359821/batch/1391151824625629225/test/lib/yuilib/3.13.0/console/lang/console_en.js TypeError: Cannot read property '13911518246308359821' of null

### ISSUE THREAD

**Issue by andrewnicols** [yui/yeti#74]
When running a server with:

```
yeti --wd-url http://localhost:4444 --server --loglevel debug
```

And then running tests with:

```
yeti --browser chrome --browser firefox mod/quiz/yui/src/autosave/tests/unit/autosave.html
```

Sometimes we see the following output:

```
[Yeti Hub] [o] [Fri Jan 31 2014 15:03:48 GMT+0800 (WST)] Unable to serve /agent/13911518246308359821/batch/1391151824625629225/test/mod/quiz/yui/src/autosave/tests/unit/assets/moodle-mod_quiz-autosave-test.js TypeError: Cannot read property '13911518246308359821' of null
    at Target.getAgent (/usr/local/lib/node_modules/yeti/lib/hub/target.js:98:23)
    at Batch.getAgent (/usr/local/lib/node_modules/yeti/lib/hub/batch.js:303:40)
    at Batch.handleFileRequest (/usr/local/lib/node_modules/yeti/lib/hub/batch.js:318:23)
    at Hub._handleTestRequest (/usr/local/lib/node_modules/yeti/lib/hub/index.js:249:15)
    at Object.testRequest (/usr/local/lib/node_modules/yeti/lib/hub/index.js:414:14)
    at nextRoute (/usr/local/lib/node_modules/yeti/lib/hub/http/router.js:116:27)
    at Router.proto.dispatch (/usr/local/lib/node_modules/yeti/lib/hub/http/router.js:121:9)
    at yetiRouter (/usr/local/lib/node_modules/yeti/lib/hub/index.js:299:28)
    at nextLayer (/usr/local/lib/node_modules/yeti/lib/hub/http/[Yeti Hub] [debug] [Fri Jan 31 2014 15:03:48 GMT+0800 (WST)] [SimpleEvents] onClose: removing data listener
```

I'm seeing this at other times too, and I'm having difficulty writing a testcase that easily reproduces this. It looks like some requests come through after an test has completed and the agentId disposed of. As a result, the this.agents list is null, and the test bombs out.

This _only_ seems to happen with firefox, and from what I've seen, only seems to happen with two or more browsers running.

**Comment by andrewnicols** [comment]
I meant to add, at the time that this happens, we get the following displayed in Firefox:

```
Internal Server Error
undefined
```

**Comment by reid** [comment]
Thanks for this report, Andrew! I'll check this out. I appreciate these details a lot!

**Comment by andrewnicols** [comment]
If I can provide any further info, let me know.

**Comment by reid** [comment]
Fixed in a9528c26d1ecb7b54cedae2311fc66834d9d62c0. Thanks for the report!

(That commit message's reference to issue 79 is incorrect. I've added a [note](https://github.com/yui/yeti/commit/a9528c26d1ecb7b54cedae2311fc66834d9d62c0#gitnotes) to the commit with corrected information.)
